docs/www/kubernetes-connectivity.md

```diff
@@ -0,0 +1,343 @@
+---
+title: Configuring Kubernetes for connectivity
+order: 0
+---
+
+The Redpanda operator supports configuration for internal and external connectivity.
```

Comment:
`operator` is technically capitalized `Operator`  as it is a k8s framework.

Reply:
This is connected to an editorial decision to not capitalize component names, like cluster and namespace, as opposed to product names, like Redpanda. CNCF capitalizes Operator as a component name but RedHat doesn't.

I would prefer to maintain our convention for caps because I think it makes the content more readable to have less caps.